x-pack/test/functional/es_archives/dashboard/feature_controls/spaces/data.json

```diff
@@ -0,0 +1,127 @@
+{
```

Comment:
What's the benefit of having different es archives for the spaces FC tests vs the security FC tests? It looks like `spaces/data.json` is a superset of `security/data.json`, so would the security tests be able to run under this data set too?

Reply:
It felt more "correct" to have each test suite only load the actual data that they need for their tests.